lib/jison.js

```diff
@@ -1280,7 +1280,12 @@ function commonjsMain (args) {
         console.log('Usage: '+args[0]+' FILE');
         process.exit(1);
     }
-    var source = require('fs').readFileSync(require('path').normalize(args[1]), "utf8");
+    var source = null;
```

Comment:
Nope, it can't. Expected input is string, always.  (... I now see you tweaked for that already.)